cpp/tests/json/json_tests.cpp

```diff
@@ -1018,4 +1018,31 @@ TEST_F(JsonPathTests, MissingFieldsAsNulls)
   do_test("$.tup[*].a.x", "[\"5\"]", "[null,null,null,\"5\"]");
 }
 
+TEST_F(JsonPathTests, QueriesContainingQuotes)
```

Comment:
Please also add a test for the invalid path.

Reply:
Thanks for pointing it out! These tests are included here https://github.com/rapidsai/cudf/blob/31506768ff1036d1971a097826229aa49e939c18/cpp/tests/json/json_tests.cpp#L565